### PR TITLE
Validate: Allow Jira issue references

### PR DESCRIPTION
**Pull request by arschmitz**
Fixes gh-45

### CODE REVIEW

File: lib/validate.js [hunk 1]
```diff
@@ -64,9 +64,9 @@ module.exports = function( message, options ) {
 		// Ticket references
 		if ( index > 0 && /^(clos|fix|resolv)(e[sd]|ing)/i.test( line ) ) {
 			if ( !/^(Fixes|Closes)\s+[^\s\d]+(\s|$)/.test( line ) &&
-					!/^(Fixes|Closes) (.*#|gh-)[0-9]+/.test( line ) ) {
+					!/^(Fixes|Closes) (.*#|gh-|[A-Z]{3}-)[0-9]+/.test( line ) ) {
 				errors.push( "Invalid ticket reference, must be " +
-					"/(Fixes|Closes) (.*#|gh-)[0-9]+/, was: " + line );
+					"/(Fixes|Closes) (.*#|gh-|[A-Z]{3}-)[0-9]+/, was: " + line );
```
Comment (zxqfox) [inline lib/validate.js]: Why only `{3}`? ;-) I'd say `{2,}`.

**Comment by arschmitz** [comment]
@zxqfox you were correct updated to match

**Comment by jzaefferer** [comment]
Thanks! Was there anything else you wanted to get in? Otherwise I'll put this into a minor release.

**Comment by arschmitz** [comment]
Not sure yet I need to test how this works with jira smart commits still.

**Comment by jzaefferer** [comment]
Ping @arschmitz - I still haven't put this into a release, waiting for you to confirm it actually works (or whatever you were going to check).

**Comment by arschmitz** [comment]
Yes this is all set i got distracted by #47

**Comment by jzaefferer** [comment]
Alright. Published as 2.3.0.

**Pull request closed without merging**